examples/finetune_with_azureml/train.yaml

```diff
@@ -0,0 +1,38 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+command: >-
+ uv run --no-sync train.py pi0_fast_libero 
+ --exp-name=${{inputs.pi0_experiment_name}} 
+ --assets-base-dir ${{outputs.ASSETS_BASE}} 
+ --checkpoint-base-dir ${{outputs.MODEL_CHECKPOINTS}} 
+ --num-train-steps ${{inputs.num_train_steps}}
+ --num-workers 8 
+ --fsdp-devices 1 
+ --batch-size 32 
+ --mlflow-enabled
+ --no-wandb-enabled
+ ${{inputs.extra_flags}}
+environment: azureml:<env_name>:<env_version>
+inputs:
+  pi0_experiment_name: <pi0_exp_name>
```

Comment:
Would it make sense to take the value of `pi0_experiment_name` from L36 `experiment_name: p0_libero_finetune`?